prerequisites/software.rst

```diff
@@ -289,6 +289,10 @@ An Elasticsearch plugin is required to index the contents of email attachments:
 
    We consider this topic as :ref:`performance_tuning`.
 
+.. hint:: **Redis will become a regular dependency in Zammad 6**
```

Comment:
Generally looking good to me.

Just a suggestion:
I'd suggest to move the whole hint block from line `292` below line `307` with the same indent.
I think it makes it much more visible that it doesn't affect memcache (even though we mention it)

![image](https://user-images.githubusercontent.com/6549061/214360821-d58eeaeb-7225-4801-b0a8-8ac2d511da83.png)

vs

![image](https://user-images.githubusercontent.com/6549061/214360963-9f3105e7-51e6-4af7-acc2-9869c77b901e.png)

Reply:
Feel free to change it this way and merge it, please!